### PR TITLE
fix typo; add testing commands

### PR DESCRIPTION
**Pull request by Eviato**
Fix getting-started issue

### CODE REVIEW

File: content/docs/developing-operators.md [hunk 1]
```diff
@@ -86,6 +86,22 @@ cd kudo
 kubectl kudo install ./config/samples/first-operator
 ```
 
+In order to see what's happen in your cluster you chan run the following command:
```
Comment (alenkacz) [inline content/docs/developing-operators.md]: chan -> can

**Review by alenkacz — APPROVED** [review]
👏 thanks!

Once we fix that one typo it's ready to go

File: content/docs/developing-operators.md (hunk 1 above)
Comment (Eviato) [reply content/docs/developing-operators.md]: Done.

**Review by meichstedt — APPROVED** [review]
thanks @Eviato!

**Pull request merged**